docs/contributing/getting-started.md

```diff
@@ -51,7 +51,7 @@ Note, how your provider doesn't need to know anything about where the DNS record
 
 # Running Github Actions locally
 
-You can also extend the CI workflow which is currently implemented as Github Action within the [workflow](../../.github/workflow) folder.
+You can also extend the CI workflow which is currently implemented as Github Action within the [workflow](https://github.com/kubernetes-sigs/external-dns/blob/master/.github/workflow) folder.
```

Comment:
This is the right link:

```suggestion
You can also extend the CI workflow which is currently implemented as Github Action within the [workflow](https://github.com/kubernetes-sigs/external-dns/tree/HEAD/.github/workflows) folder.
```

Reply:
@Raffo Thanks for review comments.
Updated with right link.